app/cinema/hall.py

```diff
@@ -0,0 +1,19 @@
+from app.people.cinema_staff import Cleaner
+from app.people.customer import Customer
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: Customer,
```

Comment:
You should use annotation like this.

``
```suggestion
            customers: list[Customer],
```

Reply:
Done